src/five/formlib/tests/forms.txt

```diff
@@ -199,20 +195,6 @@ the content, this means that no IObjectModified event should have been fired
 Unicode-safety of forms
 -----------------------
 
-To test unicode values in both Python 2 and Python 3, we need
-an auxiliary function.
-
-The ``http`` and ``http_request`` functions below expect as input
-utf-8 encoded binary strings for Python 2 and strings
-for Python 3. The following function converts our input values
-accordingly:
-
-  >>> def to_input(v):
-  ...     if six.PY2:
-  ...         return v.encode("utf-8")
-  ...     else:
-  ...         return v
-
 Even though ZPublisher does not support unicode, automatically
```

Comment:
This is no longer true: with Python 3, `ZPublisher` works with `str` (which is unicode), decoded under the assumption that `default_encoding` is the likely encoding (which can be changed with an encoding modifier).